src/App.js

```diff
@@ -64,8 +64,12 @@ export class App extends Component {
         <div className='flex flex-row-reverse-l flex-column-reverse justify-end justify-start-l' style={{ minHeight: '100vh' }}>
           <div className='flex-auto-l'>
             <div className='flex items-center ph3 ph4-l' style={{ WebkitAppRegion: 'drag', height: 75, background: '#F0F6FA', paddingTop: '20px', paddingBottom: '15px' }}>
-              <div className='joyride-app-explore' style={{ width: 560 }}>
-                <FilesExploreForm onBrowse={doFilesNavigateTo} onInspect={doExploreUserProvidedPath} />
+
+            <div className='joyride-app-explore' style={{ width: 560 }}>
+                {url.startsWith('/search')
+                  ? <IPFSSearchForm ipfsSearch={doFetchSearchResults} />
+                  : <FilesExploreForm onBrowse={doFilesNavigateTo} onInspect={doExploreUserProvidedPath} />
+                }
```

Comment:
this was used to remove the 'explore form' from the search page so that both forms were not displayed.  I agree there would need to be some thought on where the search input exists